mail/models.py

```diff
@@ -0,0 +1,40 @@
+"""
+Models for the mail app
+"""
+from django.contrib.auth.models import User
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from klasses.models import Klass
+
+
+class AutomaticReminderEmail(models.Model):
+    """
+    Stores information for an automatically sent email
+    """
+    PAYMENT_APPROACHING = 'payment_approaching'
+    REMINDER_TYPES = [PAYMENT_APPROACHING]
+    reminder_type = models.CharField(
+        choices=[(reminder_type, reminder_type) for reminder_type in REMINDER_TYPES],
+        default=PAYMENT_APPROACHING,
+        max_length=30,
+        db_index=True,
```

Comment:
Low cardinality fields like this should not have indexes: https://www.ibm.com/developerworks/data/library/techarticle/dm-1309cardinal/